pxb/v2/local/build-binary

```diff
@@ -98,6 +98,8 @@ fi
 if [[ "${DOCKER_OS}" = "centos-8" ]]; then
   if [[ ${XB_VERSION_MAJOR} -eq 8 ]] && [[ -f /opt/rh/gcc-toolset-10/enable ]]; then
     source /opt/rh/gcc-toolset-10/enable
+    # Temporary patch for PXB-3192
+    sudo yum install -y gcc-toolset-10-annobin
```

Comment:
maybe you should move install command before sourcing toolset?

Reply:
Fixed